main.js

```diff
@@ -99,6 +99,8 @@ ODate.prototype.update = function() {
 		dateString = ODate.format(date, 'date');
 	} else if (format === 'time-ago-limit-4-hours'){
 		dateString = ODate.timeAgo(date, { limit: 4*inSeconds.hour });
+	} else if (format !== '') {
```

Comment:
Agree with @roland-vachter on this, I think all we should do here is check that it is not `null` and pass it through to `ODate.format`.